premake

```diff
@@ -1,67 +1,72 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
All of the core suite premake files use `sh` instead of `bash`, although I'm not entirely sure why that is.